politeiawww/cmd/cmswww/cmswww.go

```diff
@@ -72,7 +72,7 @@ type cmswww struct {
 	LineItemPayouts     LineItemPayoutsCmd       `command:"lineitempayouts" description:"(admin)  generate line item list for a given date range"`
 	Login               shared.LoginCmd          `command:"login" description:"(public) login to Politeia"`
 	Logout              shared.LogoutCmd         `command:"logout" description:"(public) logout of Politeia"`
-	ManageUser          shared.ManageUserCmd     `command:"manageuser" description:"(admin)  edit certain properties of the specified user"`
```

Comment:
Since this is no longer a shared command, can you move `cmd/shared/manageuser.go` to `cmd/piwww/manageuser.go`.

Reply:
oh right good point